applications/DEMApplication/python_scripts/dem_default_input_parameters.py

```diff
@@ -77,6 +77,14 @@ def GetDefaultInputParameters():
             "CoordinationNumber"               : 10,
             "LocalCoordinationNumberOption"    : false,
             "GlobalCoordinationNumberOption"    : true,
+            "test_number" : 1,
```

Comment:
Everything related to SP must be in a different JSON

Reply:
Done!